app/src/components/emailVerification/NavigationFooter.tsx

```diff
@@ -60,75 +72,61 @@ export default function NavigationFooter(props: propsType): JSX.Element {
       email: formData.email,
       password: formData.password,
       token: token,
-      role: formData.role
+      role: formData.role,
     })
       .then(() => {
         AuthServices.postUserLogin({
           isContentCreator: true,
           email: formData.email,
-          password: formData.password,})
-          .then((res) => {
-              if(res.status == 200){
-              const id = res.data.baseUser;
-              navigate(`/application/${id}`);         
-              }
-              if(res.status == 202){
-                localStorage.setItem("token", res.data.accessToken);
-                localStorage.setItem("id", res.data.userInfo.id);
-                setUserInfo(res.data.userInfo);
-                navigate("/courses");
-              }
-             
-          // error messages for email and password  
-          })
+          password: formData.password,
+        }).then((res) => {
+          if (res.status === 200) {
+            const id = res.data.baseUser;
+            navigate(`/application/${id}`);
+          }
+          if (res.status === 202) {
+            localStorage.setItem("token", res.data.accessToken);
+            localStorage.setItem("id", res.data.userInfo.id);
+            setUserInfo(res.data.userInfo);
+            navigate("/courses");
+          }
+        });
       })
       .catch((err) => {
+        console.log(err);
         setError(err);
-        if (!err.response?.data) {
-          console.log(err);
-        } else {
-          switch (err.response.data.error.code) {
-            case "E0201": // Email already exists
-              break;
-            case "E0105": // Password mismatch
-              break;
-            default:
-              console.log(error);
-          }
-        }
+        toast.error(err.response?.data.message);
       });
   };
 
   return (
-    <div className=''>
-      <label>
-        <button
-          id="continue"
-          onClick={onSubmit1} // Call onSubmit when clicked
-          className="py-2 px-7 mt-8 bg-primary hover:bg-gray-100 border border-primary hover:text-primary text-white w-full transition ease-in duration-200 text-center text-base font-semibold shadow-md focus:ring-offset-2  rounded"
-        >
-          {!props.codeVerified ? 'Continuar' : 'Redefinir senha'} {/* Continue or reset password */}
-        </button>
-      </label>
-  
-      <label className="flex justify-center w-full mt-4">
-  <div className="flex items-center gap-2">
-    <p>O código não chegou?</p>
+    <div className="">
```

Comment:
A bit weird with an empty string in this className ;)

Reply:
Very Sharp eyes XD